docs/advanced/peering/peering-via-cr.md

```diff
@@ -0,0 +1,428 @@
+To do so, you should create a `Secret` containing the kubeconfig with the credentials to operate on the cluster provider, having the following labels:
+
+```{code} yaml
+liqo.io/identity-type: ControlPlane
+liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+```
+
+and annotation:
+
+```{code} yaml
+liqo.io/remote-tenant-namespace: <PROVIDER_TENANT_NAMESPACE>
+```
+
+Where the `PROVIDER_TENANT_NAMESPACE` is the tenant namespace that we created on the cluster provider for the peering with this consumer.
+
+The following is an example of identity secret:
+
+```{code} yaml
+apiVersion: v1
+data:
+  kubeconfig: <BASE64_KUBECONFIG>
+kind: Secret
+metadata:
+  labels:
+    liqo.io/identity-type: ControlPlane
+    liqo.io/remote-cluster-id: <PROVIDER_CLUSTER_ID>
+  annotations:
+    liqo.io/remote-tenant-namespace: <PROVIDER_TENANT_NAMESPACE>
+  name: cplane-secret
+  namespace: <TENANT_NAMESPACE>
+```
+
+Once you create this secret, the `liqo-crd-replicator` starts the replication of the resources, and enables the creation of `ResourceSlice` resources targetting the provider cluster. This allows the consumer to start negotiating resources with the provider cluster.
+
+After the secret creation, the logs of the `liqo-crd-replicator` should contain the following records:
+
+```text
+ k logs -n liqo liqo-crd-replicator-68f9f55dfc-bc4l8 --tail 5
+
+I1107 10:05:29.249872       1 crdReplicator-operator.go:94] Processing Secret "liqo-tenant-cl02/kubeconfig-controlplane-cl02"
+I1107 10:05:29.254977       1 reflector.go:90] [cl02] Starting reflection towards remote cluster
+I1107 10:05:29.255001       1 reflector.go:131] [cl02] Starting reflection of offloading.liqo.io/v1beta1, Resource=namespacemaps
+I1107 10:05:29.255035       1 reflector.go:131] [cl02] Starting reflection of authentication.liqo.io/v1beta1, Resource=resourceslices
+I1107 10:05:29.355741       1 reflector.go:163] [cl02] Reflection of authentication.liqo.io/v1beta1, Resource=resourceslices correctly started
+```
+
+### Summary of authentication configuration
+
+To sum up, to set up the authentication, **on the cluster provider** you will need to:
+
+- Create the credentials to be used by the consumer
+- Bind the credentials to the `liqo-remote-controlplane` role
+- Create a `Tenant` resource for the consumer
+
+While, **on the consumer side**:
+
+- Create a new secret containing the kubeconfig with the credentials to access to the cluster provider
+
+## Declarative configuration of namespace offloading
+
+While offloading is independent from the network, which means that it is possible to negotiate resources and configure a namespace offloading without the inter-cluster network enabled, **a [working authentication configuration](#declarative-configuration-of-clusters-authentication) is a pre-requisite to enable offloading**.
+
+### Ask for resources: configure a ResourceSlice
+
+The `ResourceSlice` resource is the one to be created to ask a cluster provider for resources.
```

Comment:
```suggestion
The `ResourceSlice` resource is the CR that defines the resources requested to the provider cluster.
```

Reply:
Not clear where the ResourceSlice has to be created: consumer or provider cluster?